### PR TITLE
Protect: Animate Illustration

### PR DESCRIPTION
**Pull request by nateweller**
## Proposed changes:

* Adds an animation effect to the "scan in progress" placeholder image for users without `reduced-motion` enabled.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Review the Storybook entry: `cd projects/js-packages/storybook && npm run storybook:dev`
* Review the animated image in Protect by starting a scan.

## Screenshots

animation

### CODE REVIEW

File: projects/plugins/protect/changelog/add-protect-in-progress-animation [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+Adds an InProgressAnimation component
```
Comment (nateweller) [inline projects/plugins/protect/changelog/add-protect-in-progress-animation]:
```suggestion
Significance: patch
Type: changed

Animated the "scan in progress" placeholder image
```

**Review by dkmyta — CHANGES_REQUESTED** [review]
Super nice addition!

We still face some issues with the padding of the `ScanningAdminSectionHero` and the size of the illustration when compared to the designs. Technically the padding of the `Container` is correct at `56px` but because the `InProgressAnimation` SVG includes some internal padding of its own the spacing gets thrown off a little. 

Images for reference:
Screen Shot on 2024-10-18 at 14-32-25

Screen Shot on 2024-10-18 at 14-34-08

**Comment by nateweller** [comment]
@dkmyta I've updated the PR to apply a vertical padding of `32px` to the header, and ensure the width of the graphic matches the `270px` from the "Protect Meets Core" i1 designs (since the SVG has padding to accommodate the box shadows, the image width is set to `330px`).

Screenshot 2024-10-25 at 6 20 15 PM

**Review by dkmyta — APPROVED** [review]
Looks awesome!

**Pull request merged**